website/client/src/assets/scss/typography.scss

```diff
@@ -38,6 +38,11 @@ a, a:not([href]):not([tabindex]) {
   }
 }
 
+blockquote {
```

Comment:
thanks for adding it here as well but given that right now we don't have any blockquote element and this risks getting out of sync with the markdown css, could you remove it?

Reply:
...I'm confused. The blockquote code added here is the only thing that makes the blockquote element have styling in chat posts, so removing it will make blockquotes work on tasks, guild/challenge sidebars, etc (since they use the `.markdown` class), but be unstyled/look like plain text in chat (since it doesn't use the `.markdown` class). I didn't want to start trying to add CSS classes to chat posts without knowing if it was desired (particularly since there are a couple things that are in both files but different between them), so my original changes were to the typography.scss since the Zeplin example was of styling on a chat post. Is there a different file that should be changed instead to get that styling in chat posts that I just didn't think of?

(At any rate, I may not be able to make any changes *today* specifically since it's my birthday and I'm busy enough with birthday stuff somehow that I've been interrupted five times just trying to write this reply! 😲 But I wanted to reply right now because of my confusion, instead of putting it off and eventually forgetting.)

Reply:
Hmm, sorry I got confused, I didn't realize the chat posts don't have the `markdown` class applied, I thought it was added to typography.css only as a possible future use...

@Tressley is it the expected behavior that chat posts don't get the markdown specific css styling? or maybe it's a bug we didn't notice

Reply:
For what it's worth, if I remove it from markdown.scss it does still work everywhere--that was the one I added for possible future use/differentiation.

Reply:
We pretty much want Markdown supported anywhere a user has a text input. There are some exceptions, e.g. We want to limit what Markdown is supported in the task title and checklist items to prevent the task layout from breaking.

Additionally, chats/tasks have a slightly different type scale for Markdown to prevent the task layout from breaking. Can I see an example of a task with nested quotes?

Reply:
Sorry what I meant is that the chat messages (not sure if it's a recent change or has always been the case) don't get the `markdown` CSS class applied and so they don't get the markdown specific css styles we have everywhere else. The css applied to the markdown everywhere except for the chat message is here https://github.com/HabitRPG/habitica/blob/develop/website/client/src/assets/scss/markdown.scss

Reply:
I went ahead and fired up my local install to get images of tasks (and guild/challenge descriptions... and profiles, which I forgot could hold text for a hot second) anyway. (All three use the `markdown` class, for reference.)